src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tr');
+
+rows.forEach((row) => {
+  const cells = row.querySelector('td, th');
+
+  if (cells.length > 2) {
+    const clonedCells = cells[1].cloneNode(true);
+
+    row.inserBefore(clonedCells, cells[cells.length - 1]);
```

Comment:
There is a typo in `inserBefore`. It should be `insertBefore` to correctly insert the cloned cell.